src/layerstack.stanza

```diff
@@ -236,12 +247,25 @@ public defn LayerSpec (
 
 doc: \<DOC>
 Create a Copper layer
+  Usage: Copper(name, thickness) : specify the thickness directly
+         Copper(name, weight, weight? = true) : specify weight in order to compute thickness
+         Copper(weight, AluminumMaterial, weight? = true) : without name
 <DOC>
-public defn Copper (name:String, weight:Double, material:ConductorMaterial = CopperMaterial) -> LayerSpec :
-  LayerSpec(name = name, thickness = compute-thickness(material, weight), material = material)
-
-public defn Copper (weight:Double, material:ConductorMaterial = CopperMaterial) -> LayerSpec :
-  LayerSpec(thickness = compute-thickness(material, weight), material = material)
+public defn Copper (name:String, thickness:Double, material:ConductorMaterial = CopperMaterial
```

Comment:
I'm  going to propose that we simplify this and remove the `weight` argument completely:

```
public defn Copper (thickness:Double, material:ConductorMaterial = CopperMaterial -- name:String = ?) -> LayerSpec:
   ...

```

The `weight` argument doesn't work well because there does not seem to be an established, consistent density for the copper foil applied to boards. Depending on what value is chosen for the density we get different thicknesses. I think we should simplify to just a thickness in mm.

Reply:
Replacing them with this:
```
#for (
  func-name = [Copper FR4 Soldermask]
  material-type = [CopperMaterial DielectricMaterial DielectricMaterial ]
  material-default = [CopperMaterial FR4-Material SoldermaskMaterial]
) :
  public defn func-name (thickness:Double, material:material-type = material-default
    -- name:String = ?) -> LayerSpec :
  LayerSpec(name = name, thickness = thickness, material = material)
```